components/datepicker/stories/datepicker.mdx

```diff
@@ -0,0 +1,74 @@
+import { Canvas, ArgTypes, Meta, Description, Title } from '@storybook/blocks';
+
+import * as DatePickerStories from './datepicker.stories';
+
+<Meta of={DatePickerStories} title="Docs" />
+
+<Title of={DatePickerStories} />
+<Description of={DatePickerStories} />
+
+## Usage notes
+### General notes
+- Date picker uses `.spectrum-PickerButton` instead of a `.spectrum-Picker`.
+- Workflow icon size is medium. If your markup is still using the small icon, replace `.spectrum-Icon--sizeS` with `.spectrum-Icon--sizeM`.
+
+### Component separated from InputGroup
+This component was previously a variant within InputGroup. **InputGroup is deprecated**.
+
+These classes containing `InputGroup` have been renamed
```

Comment:
Let's add a colon at the end of this line- "...have been renamed:"

Reply:
```suggestion
These classes containing `InputGroup` have been renamed:
```